webpack.dev.js

```diff
@@ -0,0 +1,30 @@
+const merge = require('webpack-merge');
+
+const common = require('./webpack.common.js');
+
+module.exports = merge(common, {
+  mode: "development",
+  
+  // Enable sourcemaps for debugging webpack's output.
+  devtool: "inline-source-map",
+  
+  entry: {
+    app: [
+      "./src/stylesheets/app.scss",
+      "./src/index.tsx"
+    ]
+  },
+
+  module: {
+    rules: [
+      {
+        test: /\.scss$/,
+        use: [
+          "style-loader",
```

Comment:
Doing this removes the the need to import `reusable-components.css` in the demo. For the dev mode, it's fine to include the css _inside_ the js file. In production the recommended way is to have its own css file. We are not using the webpack dev server (which we should use) but doing this further makes using the dev server easier.